zappa/handler.py

```diff
@@ -474,8 +473,7 @@ def handler(self, event, context):
             app_function = self.import_module_and_get_function(whole_function)
             if app_function:
                 result = self.run_function(app_function, event, context)
-                logger.debug("Result of %s:" % whole_function)
-                logger.debug(result)
+                logger.debug("Result of %s: %s", whole_function, result)
```

Comment:
No need to combine the result and the message, sometimes we want this output separate.

Reply:
If you image a cloud environment with lots of lambdas running in parallel, it can be difficult to connect isolated logger.debug(result) log prints to the correct context. What would be the reason for wanting the result in a separate log message?

If you separate them, you could end up with logs looking e.g. like this:

> Result of a
> Result of a
> Result of b
> True
> Result of a
> True
> False
> Result of c
> True
> False